src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1 +1,12 @@
-export const CommentInfo = () => <>Put the comment here</>;
+export const CommentInfo = ({ comment }) => (
```

Comment:
Ensure that the `comment` prop is passed correctly to the `CommentInfo` component. It should be an object containing `name`, `email`, and `body` properties.